packages/swarm-docs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@meetup/swarm-docs",
-  "description": "Swarm Docs",
+  "description": "Swarm Design System",
```

Comment:
since this is specifically the SDS _docs_ package, let's put 'docs' or 'public documentation site' in the description